com.kms.cura_server/src/com/kms/cura_server/UserAPI.java

```diff
@@ -19,10 +20,14 @@
 import com.kms.cura.entity.json.JsonToEntityConverter;
 import com.kms.cura.entity.user.DoctorUserEntity;
 import com.kms.cura.entity.user.PatientUserEntity;
+import com.kms.cura.entity.user.UserEntity;
 import com.kms.cura_server.resources.Strings;
 
 @Path("/user")
 public final class UserAPI {
+	private boolean statusKey = true;
```

Comment:
Yes, we have many kinds of error but all of them are considered as unsuccessful creating patient